src/plugins/Network.js

```diff
@@ -540,13 +540,15 @@ module.exports = class Network extends OverlayPlugin {
     }
     
     changeMobileSignalStrength() {
-        const signalStrength = MOBILE_SIGNAL_STRENGTH.find((elem) => elem.name === this.select.value);
+        const signalStrength = MOBILE_SIGNAL_STRENGTH.find((elem) => elem.name === this.selectMobileSignalStrength.value);
         if (signalStrength) {
-            // TODO update profile details 
-            console.log("Selected signalStrength:" + signalStrength.label)
+            const msgs = [];
+            msgs.push('setsignalstrength mobile ' + signalStrength.name);
+            const json = {channel: 'network_profile', messages: msgs};
+            this.instance.sendEvent(json);
         } else {
-            // hide profile details
-            this.profileDetails.classList.add('gm-hidden');
+            // TODO hide or do something?
+            console.log("Selected signalStrength not found")
```

Comment:
When you run the linter, doesn't calling the console give you a warning ? I usually add this import at the begining of the file:
```
const log = require('loglevel');
log.setDefaultLevel('debug');
```
then I log errors or debug messages through the appropriate call, like this: `log.error('my error message')`. But then again maybe using a console call is enough.